src/main/java/taub/gameoflife/GameOfLife.java

```diff
@@ -0,0 +1,85 @@
+package taub.gameoflife;
+
+import java.util.Arrays;
+
+public class GameOfLife
+{
+    private int[][] origGameBoard;
+    private int[][] newGameBoard;
+
+    public GameOfLife(int[][] board)
+    {
+        origGameBoard = board;
+        setNewGameBoard();
+    }
+
+    private void setNewGameBoard()
+    {
+        newGameBoard = new int[origGameBoard.length][origGameBoard[0].length];
+        for (int i = 0; i < origGameBoard.length; i++)
+        {
+            newGameBoard[i] = origGameBoard[i].clone();
+        }
+    }
+
+    private void setOrigGameBoard()
+    {
+        origGameBoard = new int[newGameBoard.length][newGameBoard[0].length];
+        for (int i = 0; i < newGameBoard.length; i++)
+        {
+            origGameBoard[i] = newGameBoard[i].clone();
+        }
+    }
+
+    public void nextGeneration()
+    {
+        for (int row = 0; row < origGameBoard.length; row++)
+        {
+            for (int col = 0; col < origGameBoard[row].length; col++)
+            {
+                determineLiveOrDead(row, col);
+            }
+        }
+        setOrigGameBoard();
+        System.out.println(this);
+    }
+
+   private void determineLiveOrDead(int row, int col)
+   {
+       int neighbors = isAlive(row - 1, col - 1) + isAlive(row - 1, col) + isAlive(row - 1, col + 1)
+                        + isAlive(row, col - 1) + isAlive(row, col + 1)
+                        + isAlive(row + 1, col - 1) + isAlive(row + 1, col) + isAlive(row + 1, col + 1);
+       if (neighbors == 3)
+       {
+           newGameBoard[row][col] = 1;
+       } else if (neighbors != 2)
+       {
+           newGameBoard[row][col] = 0;
+       }
```

Comment:
If the cell is alive in origGameBoard and neighbors==2, will it be alive in newGameBoard?

Reply:
Yes, it'll just stay a 1.